### DISCUSSION TITLE
Inlay hints at the end of the line

### DISCUSSION THREAD

**Discussion by OlekRia** [mrcjkb/rustaceanvim#640] [Q&A] [answered]
### Feature description

I wanna have some changes in rust analyzer like this:

inlay_hints = {
            auto = true,
            only_current_line = false,
            show_parameter_hints = true,
            parameter_hints_prefix = "<- ",
            other_hints_prefix = " => ",
            max_len_align = false, -- Disable alignment padding
            max_len_align_padding = 1,
            right_align = false,
            right_align_padding = 7,
            highlight = "Comment", -- Customize hint appearance
          },
          
As you may see I wanna change my code hints in a way to have :

let a = get_a_var(); => i32

instead of:

let a: i32 = get_a_var();

Is that possible?

**Comment by mrcjkb** [comment, marked as answer]
rustaceanvim doesn't implement inlay hints, they're a built-in Neovim feature. 
For what you're looking for, you can use [nvim-lsp-endhints](https://github.com/chrisgrieser/nvim-lsp-endhints).

**Reply by OlekRia** [reply]
Ir works and great! Thanks a lot.
